amf-webapi/shared/src/main/scala/amf/plugins/document/webapi/parser/spec/domain/OasResponseParser.scala

```diff
@@ -100,10 +100,12 @@ case class OasResponseParser(map: YMap, adopted: Response => Unit)(implicit ctx:
 
         // OAS 3.0.0
         if (ctx.syntax == Oas3Syntax) {
```

Comment:
[Out of scope] - Paraphrasing "this class is shit". The OasResponseParser needs a good refactor.